action/evm_transaction.go

```diff
@@ -39,51 +38,39 @@ type (
 	}
 )
 
-func NewEvmTx(a Action) *EvmTransaction {
```

Comment:
no longer needed
`Envelope` will become the struct/interface to pass to EVM protocol